components/Hero/Hero.js

```diff
@@ -1,14 +1,25 @@
 import React from 'react'
+import { Wave } from './Wave'
 
 export function Hero() {
   return (
-    
-    <div className="hero float-end m-4 w-100">
-      <div className="hero__content pe-4 text-end">
-        <p>Hello I{`'`}m</p>
-        <h1>Abla Lucia Gonzalez</h1>
+   <div className="content__hero--blue mb-4">
+      <div className="content__dark py-4">
+        <div className="container d-flex flex-column flex-md-row justify-content-between align-items-center">
+          <div className=" order-md-1 ">
+            <div className="hero my-4">
+              <div className=" pe-4 text-end w-100">
+                <p>Hello I{`'`}m</p>
+                <h1>Alba Lucía González</h1>
+              </div>
+            </div>
+          </div>
+          <img src='AlbaPortafolio.png' alt='Photo alba lucia gonzalez' className="hero__imagen"/>
```

Comment:
are we using english or spanish ??? Maybe best to practice in English, because I know you can ;)

hero__image

Reply:
oops! I forgot used English